sdk/core/azure-core-http-netty/src/test/java/com/azure/core/http/netty/NettyAsyncHttpClientProviderTests.java

```diff
@@ -88,9 +88,6 @@ public void optionsWithTimeouts() {
         Integer connectTimeout = (Integer) httpClient.nettyClient.configuration().options()
             .get(ChannelOption.CONNECT_TIMEOUT_MILLIS);
         assertEquals((int) expectedTimeout, connectTimeout.intValue());
-        assertEquals(expectedTimeout, httpClient.writeTimeout);
-        assertEquals(expectedTimeout, httpClient.responseTimeout);
-        assertEquals(expectedTimeout, httpClient.readTimeout);
```

Comment:
Any reason for removing these asserts?

Reply:
NettyAsyncHttpClient no longer has these properties